### ISSUE TITLE
Add queues directory

### ISSUE THREAD

**Issue by vmorkhat** [vmorkhat/java-algorithms-collections#1]
Add information about queue collections in Java as is the case with other collections

**Comment by Alwodan** [comment]
Ещё актуально?

**Comment by Linkshegelianer** [comment]
> Ещё актуально?

Да! Сори, уведомлений на тред не было

**Comment by vmorkhat** [comment]
> Ещё актуально?

Да! Сори, уведомлений на тред не было. Можешь добавлять то, что считаешь нужным
